makefiles/Makefile.docs-meta

```diff
@@ -0,0 +1,35 @@
+GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
+
+
+ifeq ($(STAGING_USERNAME),)
+	USER=$(shell whoami)
+else
+	USER=$(STAGING_USERNAME)
+endif
+
+
+PROJECT=docs-meta
+PREFIX=docs-meta
+MUT_PREFIX ?= $(PROJECT)
+REPO_DIR=$(shell pwd)
+
+
+SNOOTY_DB_USR = $(shell printenv MONGO_ATLAS_USERNAME)
+SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
+
+include ~/shared.mk
+
+.PHONY: help html stage
+
+## Builds this branch's HTML under build/<branch>/html
+html:
+	giza make html
+
+
+get-build-dependencies:
+	curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/docs-k8s-operator.yaml > ${REPO_DIR}/published-branches.yaml
```

Comment:
We're grabbing `docs-k8s-operator`'s yaml file here. Is this intentional for now? I'm unsure if we're actually using the information from the yaml file in the build anyways though.